include/CppUTestExt/MockSupport.h

```diff
@@ -135,7 +135,8 @@ class MockSupport
     MockExpectedCallComposite compositeCalls_;
     MockNamedValueComparatorsAndCopiersRepository comparatorsAndCopiersRepository_;
     MockNamedValueList data_;
-
+    SimpleString scope_;
```

Comment:
The name is already stored in the GetData. So, there shouldn't be need for another scope_ field, correct?